<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registrarUsuario</class>
 <widget class="QDialog" name="registrarUsuario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registrar Usuario</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="txtDireccion">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>190</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="deFechaNac">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>75</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>26</y>
      <width>108</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nombres :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Apellidos :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Fecha de Nacimiento :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Sueldo :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>DNI :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Genero :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Direccion :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QRadioButton" name="rbMasc">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>162</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Masculino</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNombres">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>24</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspSueldo">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>102</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <double>500.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>100.000000000000000</double>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDni">
    <property name="geometry">
     <rect>
      <x>148</x>
      <y>128</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>8</number>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbFem">
    <property name="geometry">
     <rect>
      <x>244</x>
      <y>162</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Femenino</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtApellidos">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>301</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="ctnCancelar">
   <property name="geometry">
    <rect>
     <x>414</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRegistrar">
   <property name="geometry">
    <rect>
     <x>329</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Registrar</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtNombres</tabstop>
  <tabstop>txtApellidos</tabstop>
  <tabstop>deFechaNac</tabstop>
  <tabstop>dspSueldo</tabstop>
  <tabstop>txtDni</tabstop>
  <tabstop>rbMasc</tabstop>
  <tabstop>txtDireccion</tabstop>
  <tabstop>btnRegistrar</tabstop>
  <tabstop>ctnCancelar</tabstop>
  <tabstop>rbFem</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
